app/controllers/permits_controller.rb

```diff
@@ -44,6 +44,11 @@ def load_adapter
       flash.alert = 'Adapter not found or not available'
       redirect_to(permits_path) && return
     end
+    user_country = current_user.organisation.country_id
```

Comment:
just realised it is possible to get to this point without being logged in :dragon: - we definitely must prevent that